.jenkins/pytorch/build.sh

```diff
@@ -22,12 +22,10 @@ source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 # (3) build with only MPI
 # (4) build with neither
 if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9*gcc7* ]] || [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9*gcc5* ]] || [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda10.1-* ]] || [[ "$BUILD_ENVIRONMENT" == *-trusty-py2.7.9* ]]; then
-  # TODO: move this to Docker
-  sudo apt-get -qq update
   if [[ "$BUILD_ENVIRONMENT" == *-trusty-py2.7.9* ]]; then
```

Comment:
do we still support python2? can we remove this to simplify?

Reply:
Yea I was going to go through and remove all trusty python 2 references in the code since that seemed deprecated.